File: src/main/java/de/pbauerochse/worklogviewer/fx/tabs/AllWorklogsTab.java
package de.pbauerochse.worklogviewer.fx.tabs;

import de.pbauerochse.worklogviewer.util.FormattingUtil;
import de.pbauerochse.worklogviewer.youtrack.TimeReport;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

/**
 * @author Patrick Bauerochse
 * @since 02.07.15
 */
class AllWorklogsTab extends WorklogsTab {

    private static final Logger LOGGER = LoggerFactory.getLogger(AllWorklogsTab.class);

    AllWorklogsTab() {
        super(FormattingUtil.getFormatted("view.main.tabs.all"));
    }

    void update(TimeReport report) {

    }
}

File: src/main/java/de/pbauerochse/worklogviewer/fx/tabs/AllWorklogsTab.kt
package de.pbauerochse.worklogviewer.fx.tabs

import de.pbauerochse.worklogviewer.util.FormattingUtil.getFormatted
import de.pbauerochse.worklogviewer.youtrack.TimeReport
import org.slf4j.LoggerFactory

/**
 * Tab, that shows all worklogs contained in the
 * [TimeReport], no matter what author or project
 */
internal class AllWorklogsTab : WorklogsTab(getFormatted("view.main.tabs.all")) {

    fun update(report: TimeReport) {

    }

    companion object {
        private val LOGGER = LoggerFactory.getLogger(AllWorklogsTab::class.java)
    }
}
